centreon/src/Core/ServiceGroup/Infrastructure/Repository/DbReadServiceGroupRepository.php

```diff
@@ -323,12 +323,50 @@ private function getFindServiceGroupConcatenator(array $accessGroupIds = []): Sq
                         `:db`.`servicegroup` sg
                     SQL
             )
+            ->appendJoins(
+                <<<'SQL'
+                    LEFT JOIN `:db`.acl_resources_sg_relations arsr
+                        ON sg.sg_id = arsr.sg_id
+                    LEFT JOIN `:db`.acl_resources res
+                            ON arsr.acl_res_id = res.acl_res_id
+                    SQL
+            )
             ->defineOrderBy(
                 <<<'SQL'
                     ORDER BY sg.sg_name ASC
                     SQL
             );
 
+        if (! empty($_GET['search']) && mb_strpos($_GET['search'], 'host_group_id')) {
```

Comment:
@Exodiaaaa Hi,
⚠️ For repositories to handle the search request parameter we do not use the global GET variable ⚠️ 
Please see how it is done in this example https://github.com/centreon/centreon/blob/2d3e27f9e7c06cf57af268cbb7f82dc74f9c02bc/centreon/src/Core/Host/Infrastructure/Repository/DbReadHostRepository.php#L446